hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/virtual/entities/OnDiskTokenRelSupplier.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2021-2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.services.state.virtual.entities;
+
+import com.swirlds.common.io.streams.SerializableDataInputStream;
+import com.swirlds.common.io.streams.SerializableDataOutputStream;
+import com.swirlds.jasperdb.SelfSerializableSupplier;
+import java.io.IOException;
+
+public class OnDiskTokenRelSupplier implements SelfSerializableSupplier<OnDiskTokenRel> {
+    static final long CLASS_ID = 0x0e52cff909625f55L;
+    static final int CURRENT_VERSION = 1;
+
+    @Override
+    public void deserialize(final SerializableDataInputStream in, final int version)
+            throws IOException {
+        // Nothing to do here
+    }
+
+    @Override
+    public void serialize(final SerializableDataOutputStream out) throws IOException {
+        // Nothing to do here
```

Comment:
Nothing to do for both de/serialize suggests these two methods belong in a separate interface, but I'm guessing they're not separated out for a reason